model/registry/deprecated/db.yaml

```
@@ -84,6 +84,13 @@ groups:
         brief: 'Deprecated, no general replacement at this time. For Elasticsearch, use `db.elasticsearch.node.name` instead.'
         deprecated: 'Deprecated, no general replacement at this time. For Elasticsearch, use `db.elasticsearch.node.name` instead.'
         examples: 'mysql-e26b99z.example.com'
+      - id: elasticsearch.cluster.name
+        type: string
+        stability: experimental
+        deprecated: Use `db.namespace` instead.
+        brief: >
+          Represents the identifier of an Elasticsearch cluster.
```

nit (to be consistent with other deprecated attributes): 

```suggestion
         Deprecated,  use `db.namespace` instead
```